bundles/org.eclipse.swt/Readme.Win32.md

```diff
@@ -2,77 +2,38 @@
-(Last tested on Win10 64 bit & Java 11. May 2019):
+### Building 64bit natives
+(Last tested on Win10 64 bit & Java 17. April 2024):
 
 You need to install the following on your system:  
 
 * Microsoft Visual Studio - Community and Windows 10 SDK:  (1*)
   https://visualstudio.microsoft.com/vs/community/
   Either select the components "MSVC C++-x64/x86-Buildtools" and "Windows 10 SDK"
   or the workload "Desktop development with C++" which includes the required components and some more.
+* Java 17 JDK
 * (Optional) Install Cygwin
-* Install Java 11. Oracle JDK or IBM JDK:  
-  http://www.oracle.com/technetwork/java/javase/overview/java8-2100321.html  
-  https://developer.ibm.com/javasdk/downloads/  
 
 ### Building and Testing locally
 
-In the binary git repo, in the windows project, you can build the natives via the build.xml. To do so:  
-
-* In Eclipse, in the binary project org.eclipse.swt.win32.win32.x86_64, right click on build.xml:  
-* Run As -> Ant Build ...
-* On the Targets tab check the build_libraries target  (it should be the only one check for just the natives).
-* On the JRE tab select "Run in the same JRE as the workspace"
-* On the Refresh tab check "Refresh resources upon completion" to refresh your workspace after running the build; this ensures that Eclipse will pick up the fresh binaries
-* Press the "Run" button to begin.
-* If compile fails, inspect build log for warnings, they can point out issues in your configuration.
-
-To test if you binaries actually get used, you could try to checkout an older binary repo commit with old
-bindings. Then try to run a snippet and get the 'missing natives' error. Then if you build the natives
-and the snippet(s) work, it shows that you've build them successfully.
-
-### Optional additional configuration
-* Install and configure Cygwin SSH Server on Windows. (Make sure 'openssh' package is installed).
-  See: https://docs.oracle.com/cd/E24628_01/install.121/e22624/preinstall_req_cygwin_ssh.htm#EMBSC281
-* Configure the machine for password-less SSH authentication with the Hudson machine.
-  For more details you can refer: http://users.cecs.anu.edu.au/~xzhang/pubDoc/IT/SSH%20without%20password%20from%20Windows.htm  
-  Sharing some key steps below(which I recall):
-  - Generate the 'dsa' public/private key from your "swtbuild" account from windows machine.
-  - Now login to the Hudson machine with "swtbuild" account.
-  - Copy the public keys and register then on the Hudson machine.. this should enable password-less authentication.
-  
+Import and build the SWT project `bundles/org.eclipse.swt` and `binaries/org.eclipse.swt.win32.win32.${arch}` in Eclipse.
+
+See **Building and Testing locally** in [Readme.md](Readme.md)
+
+Alternatively run the `build.bat` script under `org.eclipse.swt\Eclipse SWT PI\win32\library\` directory.
+To use the locally build natives run the script with the 'install' and `clean` arguments (`.\build.bat clean install`).
```

Comment:
Does not work for the same reasons as described for Linux (except the `OUTPUT_DIR` problem).